frontend/src/components/generalButtons/button.tsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import {Button, IconButton, Tooltip} from '@mui/material';
+import {CustomButtonProps} from '../../types';
+
```

Comment:
Rename the `button.tsx` file as `index.tsx` and `generalButtons` folder to `CustomButton`.